### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by jschr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,67 @@
...
+
+Install dependencies
+```bash
+cd electron-react-redux-boilerplate
+npm install
+```
+
+Development
+```bash
+npm run serve
```
Comment (jschr) [inline README.md]: @pronebird Do you have an opinion of keeping / changing `npm run serve`? Not really sure why I initially went with that but I use `npm start` or `npm run dev` these days. I feel like those are a little more widely adopted and  might be worth using instead. No strong preference either way over here.

**Comment by pronebird** [comment]
Ugh Squash/merge really messes up commits during merge :/

File: README.md (hunk 1 above)
Comment (pronebird) [inline README.md]: `serve` was popular long ago. I like `npm run develop` as it fully describes what is going on and less misleading.

**Comment by jschr** [comment]
Derp, my bad if I just messed up history, used to keeping the dev branch in sync with master. Hmmm, I can  recreate the dev branch and this PR? Would that fix it?

**Comment by pronebird** [comment]
No just leave it, I think we can squash/merge into master once again and then sync them up.

**Comment by jschr** [comment]
Ya that makes sense. I can do that for this pretty easily I think since all I did was update the readme. I'm assuming I'd just make a new branch off master `feat/updated_readme`, reapply changes, merge to master then close it and `develop`? 

Whatever you think is cleanest.

**Comment by jschr** [comment]
Addressed in https://github.com/jschr/electron-react-redux-boilerplate/pull/19.

Will close this PR and cleanup branches.

**Pull request closed without merging**